go/vt/sqlparser/token.go

```diff
@@ -358,7 +358,7 @@ var keywords = map[string]int{
 	"real":                REAL,
 	"references":          REFERENCES,
 	"regexp":              REGEXP,
-	"release":             UNUSED,
+	"release":             RELEASE,
```

Comment:
Don't forget to add these new guys to one of the reserved / non reserved keyword lists